### DISCUSSION TITLE
neovim: copilot  lazy load will not respect user settings

### DISCUSSION THREAD

**Discussion by askfiy** [community/community#13257] [Copilot Conversations] [answered]
I use packer to manage plugins, I set `vim.g.copilot_no_tab_map = true`
.

I found that vim.g.copilot_no_tab_map doesn't work when the lazyload copilot plugin is specified.

**Comment by tpope** [comment, marked as answer]
I would expect this not to work if you set it in `config`, as that runs after the plugin has already loaded. But it should work if you set it at the top level.
